HealthcareSystem/PatientWebApp/Controllers/ExaminationController.cs

```diff
@@ -53,6 +58,8 @@ public ActionResult GetExaminationsByPatient(string patientJmbg)
         /// </summary>
         /// <param name="examinationSearchDTO">an object need be find in the database</param>
         /// <returns>if alright returns code 200(Ok), if not 400(not found)</returns>
+        /// 
+        [Authorize(Roles = UserRoles.Patient)]
         [HttpPost("advance-search")]
         public ActionResult AdvanceSearchExaminations(ExaminationSearchDTO examinationSearchDTO)
         {
```

Comment:
This too, I think the DTO contains the jmbg, but you should get it from the Principal instead.

Reply:
Fixed.